modules/ossm-cr-example.adoc

```diff
@@ -3,26 +3,141 @@
 // * service_mesh/v2x/customizing-installation-ossm.adoc
 
 [id="ossm-cr-example_{context}"]
-= {ProductName} custom resources
+= Control plane parameters 
 
-[NOTE]
-====
-The `istio-system` project is used as an example throughout the {ProductShortName} documentation, but you can use other projects as necessary.
-====
+The following table lists the top-level parameters for the `ServiceMeshControlPlane` resource.
 
-A _custom resource_ allows you to extend the API in an {ProductName} project or cluster. When you deploy {ProductShortName} it creates a default `ServiceMeshControlPlane` that you can modify to change the project parameters.
+.`ServiceMeshControlPlane` resource parameters
+|===
+|Name |Description |Type
 
-The {ProductShortName} operator extends the API by adding the `ServiceMeshControlPlane` resource type, which enables you to create `ServiceMeshControlPlane` objects within projects. By creating a `ServiceMeshControlPlane` object, you instruct the Operator to install a {ProductShortName} control plane into the project, configured with the parameters you set in the `ServiceMeshControlPlane` object.
+|`apiVersion`
+|APIVersion defines the versioned schema of this representation of an object. Servers convert recognized schemas to the latest internal value, and may reject unrecognized values. The value for the `ServiceMeshControlPlane` version 2.0 is `maistra.io/v2`.
```

Comment:
IBM SG: may verb
Use only to indicate permission. Otherwise, use "can" to indicate ability, or use "might" to indicate possibility.
"May" is overused in technical information. In most cases, use "can" or "might" instead.

Reply:
Multiple instances in this PR.

Reply:
cr-example items will need to be addressed in a subsequent PR.